pkg/frontend/admin_openshiftcluster_resources_list.go

```diff
@@ -36,22 +28,25 @@ func (f *frontend) listAdminOpenShiftClusterResources(w http.ResponseWriter, r *
 		return
 	}
 
-	b, err := f._listAdminOpenShiftClusterResources(ctx, r)
+	b, err := f._listAdminOpenShiftClusterResources(ctx, r, log)
 	if err == nil {
 		b, err = adminJmespathFilter(b, jpath)
 	}
 
 	adminReply(log, w, nil, b, err)
 }
 
-func (f *frontend) _listAdminOpenShiftClusterResources(ctx context.Context, r *http.Request) ([]byte, error) {
+func (f *frontend) _listAdminOpenShiftClusterResources(
+	ctx context.Context, r *http.Request, log *logrus.Entry) ([]byte, error) {
```

Comment:
@nilsanderselde FWIW I'm not wild about all the newlines. From https://golang.org/doc/effective_go.html : "Go has no line length limit. Don't worry about overflowing a punched card."  In general this codebase aims to wrap full-line comments at 80 and doesn't care too much about long code lines unless they get really wild.